package.json

```
@@ -18,7 +18,7 @@
         "test:unit:ui": "npm run test:unit -- --ui --watch --open",
         "test:integration": "vitest --project integration",
         "test:integration:ui": "npm run test:integration -- --ui --watch --open",
-        "test:e2e": "playwright test",
+        "test:e2e": "bash scripts/start_mock_backend.sh -c 3002 3003 3004 && playwright test",
         "test:no-e2e": "vitest",
```

What about a fast test like in the CI
```suggestion
        "test:e2e": "bash scripts/start_mock_backend.sh -c 3002 3003 3004 && playwright test",
        "test:e2e:fast": "bash scripts/start_mock_backend.sh -c 3002 3003 3004 && playwright test --only-changed=origin/develop",
        "test:no-e2e": "vitest",
```